rules-reviewed/jboss-eap/weblogic/tests/weblogic-xml-descriptors.windup.test.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0"?>
+<ruletest id="weblogic-xml-descriptors-tests" xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+   <testDataPath>data</testDataPath>
+   <rulePath>../</rulePath>
+   <ruleset>
+      <rules>
+         <rule id="weblogic-xml-descriptors-01000-test">
+            <when>
+               <not>
+                   <or>
+                       <classification-exists classification="WebLogic EAR Application Descriptor" />    
+                       <!-- hint-exists message="weblogic-application.xml Elements mapping" /-->
```

Comment:
Probably, even the rule adds both classification and hint too, should it be tested both?